package.json

```diff
@@ -24,36 +24,34 @@
   },
   "homepage": "https://github.com/n1k0/readinglist-client",
   "dependencies": {
-    "6to5": "^3.0.11",
-    "bootstrap": "^3.3.2",
+    "babel": "^4.6.*",
+    "babel-eslint": "^1.0.*",
+    "bootstrap": "^3.3.*",
     "browserify": "^8.1.*",
     "docbrown": "^0.5.*",
-    "envify": "^3.2.0",
+    "envify": "^3.2.*",
     "gulp": "^3.8.*",
     "gulp-uglify": "^1.1.*",
     "gulp-webserver": "^0.9.*",
     "react": "0.12.2",
-    "reactify": "^1.0.*",
-    "rest": "^1.3.0",
+    "rest": "^1.3.*",
     "vinyl-source-stream": "^1.0.*",
     "watchify": "^2.2.*"
   },
   "devDependencies": {
-    "6to5ify": "^4.0.*",
+    "babelify": "^5.0.*",
     "chai": "latest",
-    "gulp-sourcemaps": "^1.3.0",
+    "eslint": "^0.15.1",
     "jsxhint": "latest",
```

Comment:
Are we still using `jsxhint` for anything or should we remove it from devDependencies?